docs/localizability.md

```diff
@@ -0,0 +1,152 @@
+# Localizability
+
+To enable localization, your strings must first be localizable.  Practically speaking, this means:
+
+* You put the string in a resource file (.resx).
+* You added necessary functional and contextual comments for the resource string.
+* You used the string resource instead of a string literal or format string literal.
+
+## Resource commenting
+
+[This internal site][0] has a good explanation of the what, why, and how of resource commenting.  In brief, resource comments can contain both:
+
+* functional comments, which enable tools to understand your intent
+* contextual comments, which enable human translators to understand your intent
+
+Both are important.
+
+For Arcade-based builds that use [OneLocBuild][1], adding resource comments can be done using the same Visual Studio Resource Editor that you use to add and modify resource strings in .resx files.
+
+Unfortunately, NuGet.Client does not use OneLocBuild, and any resource comments in its .resx files are [ignored][2].  The only way to comment resource strings is by editing .lci files in the NuGet.Build.Localization repository [here][3] using [lcxadmin.exe][4].  Having resource strings in one repository (NuGet.Client) and resource comments in another repository (NuGet.Build.Localization) is suboptimal.
```

Comment:
I had no clue this was a thing. 
So all of our "comments" are useless, and the translators have no way of seeing them?

Reply:
I think of the localization pipeline as supporting resource comments flowing in a certain way and ours don't.  I doubt that translators inspect our RESX files, but it's not impossible.  We should assume that they won't see our comments.